### PR TITLE
DHFPROD-5258:After mapping step is run, Tile title should be Explore, shows up Run tile title and icon

### PR DESCRIPTION
**Pull request by Sanjeevani19**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/pages/Browse.tsx [hunk 1]
```diff
@@ -168,7 +168,9 @@ const Browse: React.FC<Props> = ({ location }) => {
             location.state['query'])
         location.state['tableView'] ? toggleTableView(true) : toggleTableView(false);
     }
-
+    if(location.state && location.state['entityName'] && location.state['jobId']){
```
Comment (brucean52) [inline marklogic-data-hub-central/ui/src/pages/Browse.tsx:171]: Isn't this the same thing as line 178?

Comment (Sanjeevani19) [reply marklogic-data-hub-central/ui/src/pages/Browse.tsx:171]: @brucean52 if I have both entityName and jobId then only I am setting latestJobFacet, In the other scenario we are only checking for jobId, if that condition can be changed to include entityName too then i can remove this condition.

File: marklogic-data-hub-central/ui/src/pages/Run.test.tsx [hunk 2]
```diff
@@ -631,7 +631,8 @@ describe('Verify map/match/merge/master step failures in a flow', () => {
             fireEvent.click(exploreButton);
         }
         wait(() => {
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui/src/pages/Run.test.tsx:633]: For my tests that follow in DHFPROD-5047, I need an await before wait, so:

`await wait(() => {`

File: marklogic-data-hub-central/ui/src/pages/Browse.tsx (hunk 1 above)
Comment (brucean52) [reply marklogic-data-hub-central/ui/src/pages/Browse.tsx:171]: Yeah, it can be changed in the future.

**Pull request merged**